test/functional/Valhalla/src/org/openj9/test/lworld/ValueTypeTests.java

```diff
@@ -2460,6 +2460,322 @@ static public void testFlattenedFieldInitSequence() throws Throwable {
 		assertNotNull(getB.invoke(getD.invoke(containerC)));
 	}
 	
+	/*
+	 * Test use of DEFAULTVALUE for a value type class that has not been resolved.
+	 * The method is first called so that the DEFAULTVALUE will not be executed,
+	 * and the class not resolved, and then called so that the DEFAULTVALUE
+	 * and class resolution is triggered.
+	 */
+	@Test(priority=1)
+	static public void testUnresolvedDefaultValueUse() throws Throwable {
+		// Set up classes that look roughly like this:
```

Comment:
nitpick: please use the `/*` style comments

Reply:
Addressed in commit fbba19114